content/TrueCommand/TCReleaseNotes.md

```diff
@@ -18,16 +19,79 @@ aliases:
 
 | Version | Checkpoint | Scheduled Date |
 |---------|------------|----------------|
-| 2.3.0 | Code-freeze | 04 January 2023 |
-| 2.3.0 | Internal Testing | 09 January 2023 - 20 January 2023 |
-| 2.3.0 | Tag | 22 January 2023 |
-| 2.3.0 | Release | 23 January 2023 |
-| 2.3.1 | Code-freeze | 15 February 2023 |
-| 2.3.1 | Internal Testing | 20 February 2023 - 24 February 2023 |
-| 2.3.1 | Tag | 27 February 2023 |
 | 2.3.1 | Release | 28 February 2023 |
+| 2.3.1 | Code-freeze | 15 February 2023 |
+| 2.3.0 | Release | 17 January 2023 |
```

Comment:
That does not look right.

Reply:
This is what is posted on the Release screen in Jira so I'd say yes. Tim is going to review.